analyzer/codechecker_analyzer/cmd/check.py

```diff
@@ -515,9 +515,10 @@ def add_arguments_to_parser(parser):
                                   action='store',
                                   dest='ctu_ast_mode',
                                   choices=['load-from-pch', 'parse-on-demand'],
-                                  default='parse-on-demand',
+                                  default=argparse.SUPPRESS,
```

Comment:
I tried to run the following command `CodeChecker check -b "g++ main.cpp" --ctu-ast-mode parse-on-demand` but I do not see the error message like on the `CodeChecker analyze` command. It would be good to print this error message at the beginning of this command too without performing any operation.

Reply:
I have reorganized the parameter checking. Currently, an exception is thrown if CTU mode is not enabled for ctu-ast-mode. Is it OK to emit an exception, or should I modify the new action class (`CtuAstModeAction`) to use `LOG.error` and `sys.exit(1)` instead of `raise ValueError`?

Reply:
@gamesh411 It would be nicer to use `LOG.error` and `sys.exit(1)` instead of raise `ValueError`. See my comment above.